contracts/Nexus.sol

```diff
@@ -140,6 +142,14 @@ contract Nexus is INexus, EIP712, BaseAccount, ExecutionHelper, ModuleManager, U
             if (execType == EXECTYPE_DEFAULT) returnData = _executeBatch(executions);
             else if (execType == EXECTYPE_TRY) returnData = _tryExecuteBatch(executions);
             else revert UnsupportedExecType(execType);
+        } else if (callType == CALLTYPE_DELEGATECALL) {
+            // destructure executionCallData according to single exec
+            address delegate = address(uint160(bytes20(executionCalldata[0:20])));
+            bytes calldata callData = executionCalldata[20:];
```

Comment:
I'd move this into library's `decodeDelegatecall()` method

Reply:
ok let me see

Reply:
done